### DISCUSSION TITLE
- In Rest Transformer

### DISCUSSION THREAD

**Discussion by smurraystrata** [Budibase/budibase#12019] [Help] [answered]
Please forgive any incorrect terminology, but feel free to correct me. I'm trying to learn!.

I have often used the Transformer tab in Rest APIs to get nested data in a call, but I ran into a problem I haven't yet Googled with the correct terms to find the answer to.

I have a call which returns an array of ONTs under the "ranged-onts" object. I would like to set the transformer to "return data.ranged-onts" but the - is causing the error "Query Error: onts is not defined". I have attempted several ways to tell it ranged-onts is one object but have not yet found the correct format. Any direction here would be amazing!

Attempted formats so far - 
return data."ranged-onts"
return data.'ranged-onts'
return data.[ranged-onts]
return data.(ranged-onts)
return data.{ranged-onts}
return data.`ranged-onts`

**Comment by mike12345567** [comment, marked as answer]
Hi @jedman20 - you were very close, its the combination of two of them - you are trying to do `return data["ranged-onts"]` - the dash cannot be used with dot notation, so you have to use the property accessor, which is the square brackets. The property accessor requires a string of the property name you are trying to access, so this could be `"ranged-onts"` or `'ranged-onts'` or even something like:
```js
const propertyName = "ranged-onts"
return data[propertyName]
```
I hope this helps!

**Reply by smurraystrata** [reply]
Mike, you are awesome. That worked perfectly.
